addons/medical_statemachine/functions/fnc_enteredStateCardiacArrest.sqf

```diff
@@ -17,6 +17,7 @@
  */
 
 params ["_unit"];
+if (isNull _unit || {!isNil {_unit getVariable QEGVAR(medical,causeOfDeath)}}) exitWith {};
```

Comment:
Is this happening commonly? If not might be worth adding a WARNING in the code block.

Same for the rest of the occurrences of this check.

Reply:
In theory, shouldn't ever happen. Clockwork is supposed to filter out null and dead units. In practice, will happen every time conditions for #8772 trigger, which hasn't been exactly common IME, but could be relatively often for fatal injuries set to always.

Haven't dealt with WARNING yet. How would that look like?

Reply:
You use it like any other logging macro:
https://github.com/acemod/ACE3/blob/master/include/x/cba/addons/main/script_macros_common.hpp#L268
```sqf
if (isNull _unit || {!isNil {_unit getVariable QEGVAR(medical,causeOfDeath)}}) exitWith {
    WARNING_1("Unit is dead or null - %1",_unit);
};
```
if it's not an expected behaviour I thought we might log it.